### ISSUE TITLE
[Bug]: Unexpected error for creating invalid index

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#19408]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: milvus 2.1 latest
- Deployment mode(standalone or cluster): both
- SDK version(e.g. pymilvus v2.0.0rc2):2.1.3.dev1
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Unexpected error for creating invalid index
[gw2] [ 15%] FAILED testcases/test_index.py::TestIndexParams::test_index_type_invalid[index_type3] 

```
[2022-09-23T10:26:04.291Z] [2022-09-23 10:25:55 - ERROR - pymilvus.decorators]: Unexcepted error: [create_index], Invalid index_type: (1,), which must be one of: ['FLAT', 'IVF_FLAT', 'IVF_SQ8', 'IVF_PQ', 'HNSW', 'ANNOY', 'RHNSW_FLAT', 'RHNSW_PQ', 'RHNSW_SQ', 'BIN_FLAT', 'BIN_IVF_FLAT'], <Time: {'RPC start': '2022-09-23 10:25:55.774835', 'Exception': '2022-09-23 10:25:55.776086'}> (decorators.py:107)

[2022-09-23T10:26:04.291Z] [2022-09-23 10:25:55 - ERROR - ci_test]: Traceback (most recent call last):

[2022-09-23T10:26:04.291Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 22, in inner_wrapper

[2022-09-23T10:26:04.291Z]     res = func(*args, **kwargs)

[2022-09-23T10:26:04.291Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 56, in api_request

[2022-09-23T10:26:04.291Z]     return func(*arg, **kwargs)

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/orm/index.py", line 80, in __init__

[2022-09-23T10:26:04.291Z]     conn.create_index(self._collection.name, self._field_name, self._index_params, **self._kwargs)

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/decorators.py", line 108, in handler

[2022-09-23T10:26:04.291Z]     raise e

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/decorators.py", line 92, in handler

[2022-09-23T10:26:04.291Z]     return func(*args, **kwargs)

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/decorators.py", line 74, in handler

[2022-09-23T10:26:04.291Z]     raise e

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/decorators.py", line 48, in handler

[2022-09-23T10:26:04.291Z]     return func(self, *args, **kwargs)

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 535, in create_index

[2022-09-23T10:26:04.291Z]     check_index_params(params)

[2022-09-23T10:26:04.291Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 507, in check_index_params

[2022-09-23T10:26:04.291Z]     ", which must be one of: " + str(valid_index_types))

[2022-09-23T10:26:04.291Z] pymilvus.exceptions.ParamError: Invalid index_type: (1,), which must be one of: ['FLAT', 'IVF_FLAT', 'IVF_SQ8', 'IVF_PQ', 'HNSW', 'ANNOY', 'RHNSW_FLAT', 'RHNSW_PQ', 'RHNSW_SQ', 'BIN_FLAT', 'BIN_IVF_FLAT']

[2022-09-23T10:26:04.291Z]  (api_request.py:35)

[2022-09-23T10:26:04.291Z] [2022-09-23 10:25:55 - ERROR - ci_test]: (api_response) : Invalid index_type: (1,), which must be one of: ['FLAT', 'IVF_FLAT', 'IVF_SQ8', 'IVF_PQ', 'HNSW', 'ANNOY', 'RHNSW_FLAT', 'RHNSW_PQ', 'RHNSW_SQ', 'BIN_FLAT', 'BIN_IVF_FLAT'] (api_request.py:36)
```

### Expected Behavior

Unexpected error  -> milvus exception

### Steps To Reproduce

```markdown
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20HA%20CI/detail/PR-19406/1/pipeline
```

**Comment by XuanYang-cn** [comment]
milvus-io/pymilvus#1180

**Comment by binbinlv** [comment]
It is fixed in master(2.2), so close it.
